pkgs/tools/admin/rhoas/default.nix

```diff
@@ -26,9 +26,9 @@ buildGoModule rec {
 
   postInstall = ''
```

Comment:
Could you guard the shell completion code with `lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform)` - this will prevent build failures in cross compilation. You'll also need to add `stdenv` to the input arguments.